### DISCUSSION TITLE
How to modify navbar menu

### DISCUSSION THREAD

**Discussion by patrickduhaime** [chamilo/chamilo-lms#5941] [General] [answered]
Hello,

I am looking for a way to add some links in the navbar menu and it seems there is no easy way, I can choose what to show on the setting page but I do not see how to add my own links to the menu.

Best regards,

**Comment by chatonito** [comment, marked as answer]
para agregar link's en la barra principal, puedes hacer lo siguiente:
ve administracion de la plataforma y luego al "modificar página principal", en la parte de abajo hay una opción  de insertar página o enlace, que puedes poner en la barra de navegación.
saludos
atte 
dr. agustin rico guzman
webmaster Enseñanza Médica CAR

----------------------

To add links to the main bar, you can do the following:
Go to platform administration and then to "modify main page", at the bottom there is an option to insert a page or link, which you can put in the navigation bar.
greetings
atte 
Dr. agustin rico guzman
webmaster Medical Education CAR
